File: inf_bez/1_laba/laba1_f_bez/laba1/standard/lab1.py
import string

alphabet = string.ascii_lowercase
shift=3

def crypt(text):
    rez=""
    for c in text:
        if c != " ":
            i=alphabet.find(c)
            i= (i+shift+len(alphabet)) % len(alphabet)
            rez=rez+alphabet[i]
        else:
             rez=rez+" "
    return rez


def decrypt(text):
    rez=""
    for c in text:
        if c != " ":
            i=alphabet.find(c)
            i= (i-shift+len(alphabet)) % len(alphabet)
            rez=rez+alphabet[i]
        else:
             rez=rez+" "
    return rez


# in_str = "helo world"
# out_str=crypt(in_str)
# print(out_str)
# print(decrypt(out_str))

File: inf_bez/1_laba/laba1_f_bez/laba1/standard/lab1.java

public class lab1 {
	public static final String alphabet = "abcdefghijklmnopqrstuvwxyz";
	public static int shift = 3;

	private String f(int s,String text) {
		String rez = "";
		for (char c : text.toCharArray()) {
			if (c != ' ') {
				int i = alphabet.indexOf(c);
				i = (i + s*shift + alphabet.length()) % alphabet.length();
				rez = rez + alphabet.charAt(i);
			} else
				rez = rez + " ";
		}
		return rez;
	}
	
	public String crypt(String text) {
		return f(1,text);
	}

	public String decrypt(String text) {
		return f(-1,text);
	}

	public static void main(String[] args) {
		lab1 l=new lab1();
		String c_text=l.crypt("hello world");
		System.out.println(c_text);
		System.out.println(l.decrypt(c_text));
	}

}
